pkg/codegen/internal/test/testdata/functions.pp.ts

```diff
@@ -0,0 +1,7 @@
+import * as pulumi from "@pulumi/pulumi";
+
+const encoded = (new Buffer("haha business")).toString("base64");
```

Comment:
Is this the Node.js `Buffer`? Apparently, the constructor is deprecated in favor of factory APIs like `Buffer.from()`.

https://github.com/nodejs/node/blob/eaca13ddc9092e831b9b45180c6e0ee1f3292e5e/lib/buffer.js#L258-L267

```suggestion
const encoded = Buffer.from("haha business").toString("base64");
```